docs/kubectl_log.md

```diff
@@ -8,7 +8,7 @@ Print the logs for a container in a pod.
 Print the logs for a container in a pod. If the pod has only one container, the container name is optional.
 
 ```
-kubectl log [-f] POD [CONTAINER]
+kubectl log [-f] [-p] POD [CONTAINER]
```

Comment:
No difference. Its just convenience. For example docker's log command is `docker logs`.
This is not important for this PR.